geoiplookup/mmdblookup/mmdblookup_test.go

```diff
@@ -13,7 +17,20 @@ const (
 	ipAddr        = "35.204.49.125"
 )
 
+func maybeFetchResources(t *testing.T) {
```

Comment:
This test assumed other tests run before it. This new function ensures we have all fixtures always.